common/app/conf/switches.scala

```diff
@@ -363,6 +363,15 @@ object Switches {
     exposeClientSide = true
   )
 
+  val CriteoSwitch = Switch(
+    "Commercial",
+    "criteo",
+    "If this switch is on, Criteo segments will be used to target ads.",
+    safeState = Off,
+    sellByDate = never,
```

Comment:
Good catch. I've been fiddling with Git trying to revert the reverting revert commit that I missed this.